pkg/providers/aws/cluster_network_provider.go

```diff
@@ -1157,6 +1176,30 @@ func (n *NetworkProvider) reconcileRDSVpcConfiguration(ctx context.Context, priv
 	return nil
 }
 
+//this function removes tags and reads them to an rds subnet group
+func (n *NetworkProvider) updateSubnetGroupTags(foundSubnet *rds.DBSubnetGroup, croOwnerTag *rds.Tag, logger *logrus.Entry) error {
+	logger.Info()
+	_, err := n.RdsApi.RemoveTagsFromResource(&rds.RemoveTagsFromResourceInput{
+		ResourceName: aws.String(*foundSubnet.DBSubnetGroupArn),
+		TagKeys: []*string{
+			aws.String(*croOwnerTag.Key),
```

Comment:
`croOwnerTag.Key` I'm guessing is a pointer to a string, `*string`?

The `*` dereferences it to the string value

`aws#String(string) *string` takes in a string value and returns a pointer.

So if I've understood correctly, this is the same as doing:

```suggestion
			croOwnerTag.Key,
```